pages/reference/api/create-webhook.en-US.mdx

```diff
@@ -0,0 +1,59 @@
+| `recording.waiting`        | This fires after a stream with recording on has concluded and is not yet ready to be downloaded. Typically it takes 5 minutes for recordings to be ready for download. |
+| `multistream.connected`    | This fires when we've successfully connected to the multistream target                                                                                                 |
+| `multistream.error`        | This fires when we've encountered an error either while attempting to connect to the third party streaming service or while broadcasting.                              |
+| `multistream.disconnected` | This fires when we are no longer sending video to the multistream target.                                                                                              |
+| `asset.created`            | This fires when a On Demand asset is created.                                                                                                                          |
+| `asset.updated`            | This fires when a On Demand asset is updated.                                                                                                                          |
+| `asset.ready`              | This fires when a On Demand asset is ready.                                                                                                                            |
+| `asset.failed`             | This fires when a On Demand asset fails during the upload or during processing.                                                                                        |
+| `asset.deleted`            | This fires when a On Demand asset is deleted.                                                                                                                          |
+| `task.spawned`             | This fires when a task is spawned. (For example, an On Demand upload)                                                                                                  |
+| `task.updated`             | This fires when a task is updated.                                                                                                                                     |
+| `task.completed`           | This fires when a task completes its execution successfully.                                                                                                           |
+| `task.failed`              | This fires when a task has failed.                                                                                                                                     |
+| `playback.accessControl`   | Dedicated webhook for playback access control                                                                                                                          |
```

Comment:
WDYT of adding a small remark that this webhook works differently than the others? That it needs to be references in a playback policy instead of just "existing" to be called, like the others.

Reply:
Done!